zipline-loader-testing/build.gradle.kts

```diff
@@ -22,6 +21,8 @@ kotlin {
   tvosSimulatorArm64()
   tvosX64()
 
+  applyDefaultHierarchyTemplate()
+
```

Comment:
```suggestion
```
This project doesn't have any custom source sets. Shouldn't be needed explicitly.